phoenix-core/src/test/resources/hadoop-metrics2.properties

```diff
@@ -32,10 +32,9 @@
 #    [prefix].[source|sink].[instance].[options]
 # See javadoc of package-info.java for org.apache.hadoop.metrics2 for detail
 
-
-# Don't attempt to start jmx mbeans for all sources.
-#  For right now, all metrics are exported to a phoenix table
-*.source.start_mbeans=false
+hbase.source.start_mbeans=true
+hbase.sink.sink0.class=org.apache.phoenix.monitoring.GlobalPhoenixMetricsTestSink
```

Comment:
Can you leave a comment in the test that this configuration is here? Otherwise, might seem like voodoo

Reply:
Sure. It was hard for me as well since things were not working as expected in the first place. Took a little while to realize this file was the reason :)

Reply:
Haha, perfect :)